lib/location/method_gnss.c

```diff
@@ -79,18 +89,24 @@ static bool running;
 static K_SEM_DEFINE(entered_psm_mode, 0, 1);
 static K_SEM_DEFINE(entered_rrc_idle, 0, 1);
 
-#if (defined(CONFIG_NRF_CLOUD_AGPS) || defined(CONFIG_NRF_CLOUD_PGPS))
+#if defined(CONFIG_NRF_CLOUD_AGPS) || defined(CONFIG_NRF_CLOUD_PGPS)
 static struct nrf_modem_gnss_agps_data_frame agps_request;
```

Comment:
"agps_request" with CONFIG_NRF_CLOUD_PGPS?

Reply:
Same datatype is used by some of the P-GPS functions, e.g. nrf_cloud_pgps_inject() takes this parameter in.